src/test/java/teammates/test/cases/browsertests/FeedbackMsqQuestionUiTest.java

```diff
@@ -128,6 +128,37 @@ public void testInputValidation() {
         feedbackEditPage.clickAddMsqOtherOptionCheckboxForNewQuestion();
         assertTrue(feedbackEditPage.isElementSelected("msqOtherOptionFlag--1"));
         feedbackEditPage.clickAddQuestionButton();
+
+        ______TS("Check that Max/Min selectable choices cannot be blank");
+
+        feedbackEditPage.clickNewQuestionButton();
+        feedbackEditPage.selectNewQuestionType("MSQ");
+        feedbackEditPage.fillQuestionTextBoxForNewQuestion("Test question text");
+        feedbackEditPage.fillQuestionDescriptionForNewQuestion("more details");
+        feedbackEditPage.fillMsqOptionForNewQuestion(0, "Choice 1");
+        feedbackEditPage.fillMsqOptionForNewQuestion(1, "Choice 2");
+
+        feedbackEditPage.toggleMsqMaxSelectableChoices(NEW_QUESTION_INDEX);
+        feedbackEditPage.fillMsqMaxSelectableChoices(NEW_QUESTION_INDEX, "");
+        assertFalse(feedbackEditPage.isInputElementValid(
+                feedbackEditPage.getMsqMaxSelectableChoicesBoxElement(NEW_QUESTION_INDEX)));
+        feedbackEditPage.setMsqMaxSelectableChoices(NEW_QUESTION_INDEX, 2);
```

Comment:
I think just fix to using `set` or `fill`?

Reply:
I would suggest using fill.

Reply:
Okay.

Reply:
@LiHaoTan @Crphang Should I replace all occurrences of `set*` to `fill*` and remove the `setMsqMaxSelectableChoices` method?

Reply:
@Crphang I think https://github.com/TEAMMATES/teammates/pull/8678#discussion_r178341666 would be a better approach. what do you think?